src/applications/appeals/996/subtask/pages/start.jsx

```diff
@@ -73,9 +73,9 @@ const BenefitType = ({ data = {}, error, setPageData }) => {
         Is this the form I need?
       </h2>
       <p>
-        Use this form if you disagree with VA’s decision on your claim and want
-        to request that a senior reviewer take a new look at your case and the
-        evidence you provided. You can’t submit any new evidence with a
+        Use this form if you disagree with VA’s decision on your initial or
+        supplemental claim, and want to request that a new review of your case
```

Comment:
`that` seems to be incorrectly added and should be removed, or the sentence ending needs to be changed (e.g. `to request that a new review of your case is conducted by a higher-level reviewer`)

Reply:
I copy pasted from @eileen-coforma. Let's see what she recommends.

Reply:
@Mottie mentioned it's the `that` that's out of place. That is indeed a typo. It should be removed altogether

Reply:
Fixed